protocols/grpc/agent.proto

```diff
@@ -251,12 +251,13 @@ message CreateSandboxRequest {
 	string hostname = 1;
 	repeated string dns = 2;
 	repeated Storage storages = 3;
-	
+
 	// This field means that a pause process needs to be created by the
-	// agent. This pid namespace of the pause process will be treated as 
+	// agent. This pid namespace of the pause process will be treated as
 	// a shared pid namespace. All containers created will join this shared
 	// pid namespace.
 	bool sandbox_pidns = 4;
+	string sandbox_id = 5;
```

Comment:
It might be worth adding a comment here explaining that we don't need this member in the other sandbox and network calls (as listed in https://github.com/kata-containers/tests/issues/453#issuecomment-401339364) since we implicitly require `CreateSandbox()` to be called before any other sandbox/network calls :)

Reply:
Sure, I'll add some text about it.